Setting `extractCSS` to `true` cause error: `Invalid CSS after "...  var cssReload": expected selector, was '= require`

### Version

[v2.4.0](https://github.com/nuxt.js/releases/tag/v2.4.0)

### Reproduction link

[https://codesandbox.io/s/z6510o74ql?autoresize=1&hidenavigation=1&module=%2Fnuxt.config.js&moduleview=1](https://codesandbox.io/s/z6510o74ql?autoresize=1&hidenavigation=1&module=%2Fnuxt.config.js&moduleview=1)

### Steps to reproduce

- Create a new Nuxt@^2.4.0
- Add `SCSS` support `npm install --save-dev node-sass sass-loader`
- Set `extractCSS` to `true` in the `build` property of `nuxt.config.js`
- Set an external 'scss' file for 'index.vue' page

### What is expected ?

To work as expected

### What is actually happening?

Error in console and in browser:

Module build failed (from ./node_modules/extract-css-chunks-webpack-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/lib/loader.js):

undefined
                   ^
      Invalid CSS after "...  var cssReload": expected selector, was '= require("../node_'

### Additional comments?

- 2.3.4 was working ok
- Seems a problem related to 'SCSS' loader.

<!--cmty--><!--cmty_prevent_hook-->
<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c8564">#c8564</a>)</em></sub></div>

FWIW, I also get massive pathing-related errors with Nuxt 2.4.0 (and extract-css-chunks-webpack-plugin) that were not present in 2.3.4.
```
Module build failed (from ./node_modules/extract-css-chunks-webpack-plugin/dist/loader.js):
ModuleNotFoundError: Module not found: Error: Can't resolve 'fonts/myfont.woff' in '\assets\styles'
```

--- a/packages/config/src/options.js
+++ b/packages/config/src/options.js
@@ -291,6 +291,10 @@ export function getNuxtConfig(_options) {
     options.build.optimization.minimize = !options.dev
   }
 
+  if (options.dev) {
+    options.build.extractCSS = false
+  }
+
   // Enable optimizeCSS only when extractCSS is enabled
   if (options.build.optimizeCSS === undefined) {
     options.build.optimizeCSS = options.build.extractCSS ? {} : false

